src/openforms/config/admin.py

```diff
@@ -73,6 +73,16 @@ class GlobalConfigurationAdmin(
                 )
             },
         ),
+        (
+            _("Form map component options"),
+            {
+                "fields": (
+                    "form_map_default_zoom_level",
+                    "form_map_default_latitude",
+                    "form_map_default_longitude",
+                )
+            },
+        ),
```

Comment:
I'd move this under "General form options", you can provide a tuple of field names to `"fields"` so that they're all on one line to group them together.